lib/srv/regular/sshserver_test.go

```diff
@@ -1411,13 +1411,51 @@ func TestClientDisconnect(t *testing.T) {
 	require.NoError(t, clt.Close())
 }
 
+func testHelperWaitForServerStart(t *testing.T, s *Server, config *ssh.ClientConfig, timeout time.Duration) {
+	for end :=  time.After(timeout); true;{
+		select {
+		case <-end:
+			t.Fatalf("Server failed to start after: %v", timeout)
+			return
+		default:
```

Comment:
This `default` case is going to execute every time since the read from `end` will block. This is likely not what you want.

Can you rewrite this to use `require.Eventually` instead?

Reply:
Ah nice i didn't know that existed. TY this will be handy